to.etc.domui/src/main/java/to/etc/domui/util/Msgs.java

```diff
@@ -151,6 +151,9 @@
 	/** Value out of range */
 	public static String V_OUT_OF_RANGE = "v.out.of.range";
 
+	/** Invalid not regex */
+	public static String V_INVALID_REGEX = "v.invalid.regex";
```

Comment:
Why would that error be here? It is nowhere used within DomUI.
It should also be an enum constant if we do need it.

Reply:
well, why not? I mean, isn't it convenient to have it here? There are also other validation messages that we have, that seem unused in domui as well? I see that V_INVALID_DOUBLE and V_OUT_OF_RANGE are also unused?

Reply:
Because others did something wrong I'm now allowed too is never going to fly well..

It is one thing to have really common messages here. But this one is really not common...

You can leave it, but do change it to an enum field (I need to do that with all others too, and it is a shitload of work).

Reply:
Well, in my eyes it was not wrong at all. It is not that I did it because others did it before -> I did it because I think it is useful to have it as one of possible built in validation messages. But ok... So, you want me to spend a day in moving all that to an enum? It has 721 usages, everywhere... And it would also break the backward compatibility of domui used elsewhere. Do we really want that now?

Reply:
ah, ok, you meant to add that new entry as enum member... Wasn't aware that we have that partially done already here... Ofc, fixed.